project/backend/doc-server/src/main/resources/files/dd398660-ca95-4b58-97c8-5c7c5d2a8a31.py

```
@@ -0,0 +1,128 @@
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+
+load_dotenv()
+
+api_key = os.getenv('gemini_api_key')
+url = "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash-latest:generateContent"
+
+app = Flask(__name__)
+CORS(app)
+
+# Utility functions
+def split_document(doc, chunk_size=5):
```

Ruff (0.8.2)

Missing return type annotation for public function `split_document`

(ANN201)